src/components/RichEditor/RichEditorBase/RichEditorBase.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from "react";
+
+import BoxBase from "src/components/Boxs/BoxBase";
+import TypographyBase from "src/components/Typography/TypographyBase";
+
+import RichEditorBase, { RichEditorBaseProps } from ".";
+
+import { Meta, Story } from "@storybook/react";
+
+export default {
+    title: "Components/RichEditorBase",
+    component: RichEditorBase,
+} as Meta;
+
+const Template: Story<RichEditorBaseProps> = (args) => {
+    const [value, setValue] = useState<string>("");
+    const getEditorValue = (content: string) => {
+        setValue(content);
+    };
+    const { getValue, ...rest } = args;
+    return (
+        <BoxBase width={1} display="flex">
```

Comment:
What does 1 mean here?

Reply:
w={1} is same width="100%".
I had demos of width on stories about size of boxbase and wrote description in width field.
![image](https://user-images.githubusercontent.com/58205909/121801905-f342c500-cc63-11eb-9ba1-5759195c1aea.png)

Reply:
Cool